resources/css/input-number.css

```diff
@@ -0,0 +1,9 @@
+.no-arrow::-webkit-inner-spin-button,
+.no-arrow::-webkit-outer-spin-button {
+    -webkit-appearance: none !important;
+    margin: 0 !important;
+}
+
+.no-arrow {
+    -moz-appearance: textfield !important;
+}
```

Comment:
It wont take effect for the end developers, the wireui styles in only for testing, delete this file

Reply:
Done!